main/src/library/Prelude.flix

```diff
@@ -174,29 +174,28 @@ pub def stringify(x: a): String = {
         case ReifiedBool =>
             (b -> if (b) "true" else "false") as a -> String
         case ReifiedChar =>
-            import static java.lang.Character.toString(Char): String \ {};
-            toString as a -> String
+            (c -> "\'" + escape("${c: Char}") + "\'") as a -> String
```

Comment:
One question: Why are we returning a function? Would it not be better to simply return the string??

Reply:
Something weird with monomorph, I guess. @JonathanStarup may have more information.

Reply:
because we dont eliminate the reification match at compile time, the verifier complains that the irrelevant cases cast to a wrong type. ype reification is a bit limited right now as a consequence thereof